src/main/java/website/frontrow/game/Game.java

```diff
@@ -61,9 +60,29 @@ public void tick()
             {
                 levelWon();
             }
+            
+            if(playersDead())
+            {
+            	levelLost();
+            }
         }
         updateObservers();
     }
+    
+    /**
+     * This is used by just a few tests that didn't work the way they
+     * should've worked, because they used a mocked level.
+     * Because the level has no enemies, the game would stop.
+     * In order for the tests to be accurate, I made a new tick
+     * function that only checks whether or not the players are dead.
```

Comment:
Please don't do this. Do not write extra functions just for testing.
If your testing case stops because of a lack of enemies, stub the method that checks whether all enemies are dead instead.

< testing